### PR TITLE
add validator DOUBLETOP

### PR DESCRIPTION
**Pull request by kuraassh**

### CODE REVIEW

File: packages/sdk-ui-ts/src/utils/mappings.ts [hunk 1]
```diff
@@ -311,4 +311,6 @@ export const validatorAddressToPathMap = {
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
     injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj:
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp'
+  injvalcons1n5yluh6k9zqv74jw35r5s5zy808xpzfqp2wm7g:
```
Comment (bangjelkoski) [inline packages/sdk-ui-ts/src/utils/mappings.ts:314]: Missing wrapping the address in `'` here and you also miss the `injvaloper...` address mapping

**Review by bangjelkoski — CHANGES_REQUESTED** [review]

File: packages/sdk-ui-ts/src/utils/mappings.ts (hunk 1 above)
Comment (kuraassh) [reply packages/sdk-ui-ts/src/utils/mappings.ts]: updated

File: packages/sdk-ui-ts/src/utils/mappings.ts [hunk 2]
```diff
@@ -310,5 +310,7 @@ export const validatorAddressToPathMap = {
     injvalcons1m8shfyz7cu7w0umgwgmhk59nmatzlvu7g56t6a:
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
     injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj:
-    'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp'
```
Comment (bangjelkoski) [inline packages/sdk-ui-ts/src/utils/mappings.ts:313]: Why is this removed?

File: packages/sdk-ui-ts/src/utils/mappings.ts [hunk 3]
```diff
@@ -310,5 +310,7 @@ export const validatorAddressToPathMap = {
...
+    'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
+    injvalcons1n5yluh6k9zqv74jw35r5s5zy808xpzfqp2wm7g:
+    'injvaloper1n5yluh6k9zqv74jw35r5s5zy808xpzfq4ea8jf.webp'
```
Comment (bangjelkoski) [inline packages/sdk-ui-ts/src/utils/mappings.ts:315]: You are still missing one mapping. Please fix it properly before committing again.

**Review by bangjelkoski — CHANGES_REQUESTED** [review]

File: packages/sdk-ui-ts/src/utils/mappings.ts (hunk 2 above)
Comment (kuraassh) [reply packages/sdk-ui-ts/src/utils/mappings.ts:313]: I did not delete the line, this is the ATOM code editor so it pulled up and displays the change

File: packages/sdk-ui-ts/src/utils/mappings.ts (hunk 3 above)
Comment (kuraassh) [reply packages/sdk-ui-ts/src/utils/mappings.ts]: Sorry for this, I have problem with my code editor app. Fixed from github

**Review by bangjelkoski — APPROVED** [review]

**Comment by bangjelkoski** [comment]
I'll republish the packages and update our products in a bit.
